### DISCUSSION TITLE
How to list peer dependency issues?

### DISCUSSION THREAD

**Discussion by agriffis** [pnpm/pnpm#5685] [Q&A] [answered]
When I add a new package with `pnpm add` then pnpm shows me a list of peer dependency issues in the current repository.

How can I get that list again, later on? Is there a standalone command to purely list the peer dep problems?

**Comment by lvqq** [comment]
Could we add this feature to `pnpm doctor` with option such as `--peer`? @zkochan

**Reply by zkochan** [reply]
I don't think so. `pnpm doctor` is for listing issues with pnpm installation and configuration.

**Reply by lvqq** [reply]
Thanks! What's your suggestion about it? Or is it planned to be supported?

**Reply by zkochan** [reply]
Maybe the list command could be updated to list peer issues.

**Comment by lvqq** [comment, marked as answer]
In version 8.3.0, a new option has been added: `pnpm install --resolution-only`. It re-runs the resolution process to print out any peer dependency issues.

**Reply by agriffis** [reply]
Wonderful, thanks!

**Comment by otakutyrant** [comment]
What does "peer dependency issues" mean? Does `pnpm install --resolution-only` show all uninstalled peerDependencies automatically? I installed next-auth but ESLint complained that some modules are missing, after debugging I found out that it was because I did not install next-auth's peerDependencies `@auth/next`.

Now I wonder how can I determine what peerDependencies I do not install too. `pnpm install --resolution-only` seems not work because it does not show `@auth/core` which I does not install.

`pnpm view <package> peerDependencies` can show a package's peerDependencies, but it is now what I want. It takes a lot of time to try this command in every package.
